### PR TITLE
refactor(machine): improve machine marker value

### PR DESCRIPTION
**Pull request by BlackHole1**
1. Set the marker to the current virtual machine type instead of fixed qemu.
2. Update containers/common

[NO NEW TESTS NEEDED]

#### Does this PR introduce a user-facing change?

```release-note
None
```

/cc @baude

### CODE REVIEW

**Comment by mheon** [comment]
LGTM
@baude @ashley-cui PTAL

**Review by ashley-cui — COMMENTED** [review]
/lgtm

File: pkg/machine/ignition.go [hunk 1]
```diff
@@ -434,8 +434,7 @@ Delegate=memory pids cpu io
 		FileEmbedded1: FileEmbedded1{
 			Append: nil,
 			Contents: Resource{
-				// TODO this should be fixed for all vmtypes
-				Source: EncodeDataURLPtr("qemu\n"),
```
Comment (jakecorrenti) [inline pkg/machine/ignition.go:438]: Will this cause issues since applehv isn't supported yet?

**Comment by jakecorrenti** [comment]
can't give the approval, but lgtm

**Pull request merged**